lib/taurus/qt/qtgui/panel/taurusvalue.py

```diff
@@ -1212,13 +1216,31 @@ def getLabelConfig(self):
 
     @Qt.pyqtSlot('QString')
     def setLabelConfig(self, config):
+        """Sets fragment configuration to the label widget.
+
+        :param config: fragment
+        :type config: str
+        """
+        # backwards compatibility: this method used to work for setting
+        # an arbitrary text to the label widget
+        try:
+            self.getModelFragmentObj(config)
+        except Exception:
```

Comment:
Is it not possible to narrow this exception to AttributeError?

Reply:
What we get there is a simple Exception:

```
Traceback (most recent call last):
  File "/home/zreszela/workspace/taurus/lib/taurus/qt/qtgui/panel/test/test_taurusvalue.py", line 56, in test_bug126
    w.setLabelConfig(label)
  File "/home/zreszela/workspace/taurus/lib/taurus/qt/qtgui/panel/taurusvalue.py", line 1227, in setLabelConfig
    self.getModelFragmentObj(config)
  File "/home/zreszela/workspace/taurus/lib/taurus/qt/qtgui/base/taurusbase.py", line 631, in getModelFragmentObj
    return self.modelObj.getFragmentObj(fragmentName)
  File "/home/zreszela/workspace/taurus/lib/taurus/core/taurusmodel.py", line 201, in getFragmentObj
    obj = getattr(obj, fn)
  File "/home/zreszela/workspace/taurus/lib/taurus/core/tango/tangoattribute.py", line 303, in __getattr__
    % name)
Exception: TangoAttribute does not have the attribute MIXEDcase

```